### PR TITLE
CQ-4318780: Updated CQAssetsClient to return DTO

### PR DESCRIPTION
**Pull request by brandonmaynard**

### CODE REVIEW

**Review by volteanu — APPROVED** [review]
LGTM!

File: src/main/java/com/adobe/cq/testing/client/assets/dto/ProcessedAsset.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+public class ProcessedAsset {
+
+    private String assetPath;
+    private List<String> processedRenditions;
+    private List<FailedRendition> failedRenditions;
+
+    /**
+     * Constructor.
+     */
+    public ProcessedAsset() {
```
Comment (dulvac) [inline src/main/java/com/adobe/cq/testing/client/assets/dto/ProcessedAsset.java]: why create it if it does nothing? :)

**Review by dulvac — APPROVED** [review]
@brandonmaynard LGTM! 
But FYI, this would have to be a major release because it's backward-incompatible with the signature change (cc @volteanu )

**Comment by volteanu** [comment]
@brandonmaynard how soon do you need a new release with this changes? (sorry it got delayed for so long). I'm asking because we also have #53 to be merged soon and I would make only 1 release once that is done.

**Comment by brandonmaynard** [comment]
Hi @volteanu , no problem at all. It should be fine to release when #53 is ready. No huge rush

**Comment by volteanu** [comment]
Released in aem-cloud-testing-clients-0.7.0

**Comment by brandonmaynard** [comment]
Thank you @volteanu!

**Pull request merged**